src/duration.rs

```diff
@@ -444,9 +458,18 @@ fn div_rem_64(this: i64, other: i64) -> (i64, i64) {
 #[cfg(test)]
 mod tests {
     use super::{Duration, MIN, MAX, OutOfRangeError};
-    use std::{i32, i64};
+
+    #[cfg(feature = "std")]
```

Comment:
Since testing always implies the standard library, could we just always run tests with the `std` feature?